buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -466,6 +466,28 @@
     "PB_CreateHelixArguments": {
       "value": "/t:CloudBuild /p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:\"EnableCloudTest=true\" /p:\"BuildMoniker=none\" /p:\"TargetQueue=Windows.10.Amd64\" /p:\"TestProduct=corefx\" /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Windows_NT\" /p:FilterToOSGroup=Windows_NT",
       "allowOverride": true
+    },
+    "PB_OptionalToolSource": {
+      "value": null,
+      "allowOverride": true,
+      "isSecret": true
```

Comment:
Yes, these are 3 extra secret parameters that I'll need to add to the root def and pass through. The other tools (talked about them offline) seem like they can use the same flow, so we shouldn't have to add any more secrets for those.